src/modules/Lan.rb

```diff
@@ -1138,7 +1138,9 @@ def routing_summary
       config = find_config(:yast)
       return "" unless config && config.routing
       presenter = Y2Network::Presenters::RoutingSummary.new(config.routing)
-      presenter.text
+      summary = "<ul><li><b>" + _("Routing:") + "</b></li><li>"
+      summary += presenter.text
+      summary += "</li></ul>"
```

Comment:
this looks really strange. If you need specific summary for autoinst routing, please create a child of RoutingSummary and redefine text there.

Reply:
I just wanted to check how it looked like in the AutoYaST summary, as it was suggested during the network installation proposal review.

Reply:
![Proposal](https://trello-attachments.s3.amazonaws.com/5b2b92a1265a6fc3b05a7d51/5b757c37874e4c7fb27c7cf9/134fd0efcbae56bcdcc98cff9ed847dc/FullNetworkProposal.png)

Reply:
hmm, this is really big. I think there is ways to compress like having together IP Forwarding or somehow compress it with less details. This way it is a lot of info that has problem to fit and I worry in ncurses it will be even worse.

Reply:
Although this concretely presentation should be responsibility of `Yast::Lan` Summary 

https://github.com/yast/yast-network/blob/master/src/modules/Lan.rb#L813

Reply:
Yep, and get bigger with complex configuration